scripts/__init__.py

```diff
@@ -0,0 +1 @@
+from cddl_gen import PythonClass
```

Comment:
Do we need this to access the class within the same file?

Reply:
__init__.py is what makes it a module, and I read that this is how you make symbols available externally.